packages/xo-web/src/xo-app/backup/new/smart-backup.js

```diff
@@ -23,6 +26,40 @@ const VMS_STATUSES_OPTIONS = [
   { value: 'Halted', label: _('vmStateHalted') },
 ]
 
+const CustomTag = decorate([
+  provideState({
+    initialState: () => ({
+      editing: false,
+    }),
+    effects: {
+      addTag: (effects, newTag) => (_, props) =>
+        props.handler(props.data == null ? [newTag] : [...props.data, newTag]),
+      closeEdition: () => ({ editing: false }),
+      toggleState,
+    },
+  }),
+  injectState,
+  ({ state, effects }) =>
+    state.editing ? (
+      <EphemeralInput
+        closeEdition={effects.closeEdition}
+        onChange={effects.addTag}
+        type='text'
+      />
+    ) : (
+      <Tooltip content={_('customTag')}>
+        <Button name='editing' onClick={effects.toggleState} size='small'>
+          <Icon icon='edit' />
+        </Button>
+      </Tooltip>
+    ),
+])
+
+CustomTag.propTypes = {
+  data: PropTypes.array,
```

Comment:
@pdonias  `data` or `previousTags`? What do you think?

Reply:
`selectedTags`